### DISCUSSION TITLE
Tcmmrc F4129 control firm= matekf405 = MCU: STM32F405 INav 2.5.2 BUG Bluetooth and LED RGB PAt

### DISCUSSION THREAD

**Discussion by nleo22** [iNavFlight/inav#6204] [Q&A]
i got board Tcmmrc F4129 control firm= matekf405 = MCU: STM32F405   2.5.2 inav
From betaflight=4.1.1 matekf405 everything work fine. Bluetooth and rgb working fine.

Then every since I switched to Inav 2.5.2 STM3f405 bluetooth and rgb led doesn't work anymore please help.

[a comment by an automated account is not shown]

**Comment by DzikuVx** [comment]
This board is not oficially supported by INAV. Have you tried to contact the manufacturer?

**Comment by nleo22** [comment]
> This board is not oficially supported by INAV. Have you tried to contact the manufacturer?

I did but Noone answer. Even on tcmmrc Facebook 
I'm thinking to replace it. For this HGLRC Zeus F72.
Do you think this one support inav? How do you know? Where to look for...
https://www.amazon.com/gp/aw/d/B08BL5GVR6/ref=ya_aw_od_pi?ie=UTF8&psc=1
